consul/base.py

```diff
@@ -57,6 +57,7 @@ def __init__(
             host='127.0.0.1',
             port=8500,
             token=None,
+            ssl=False,
```

Comment:
How about scheme='http' instead?, Customizable to be 'https' .. not sure why I think that seems better :) Just a vibe.